README.md

```diff
@@ -84,6 +84,7 @@ Game networking is a subset of computer networking that specifically covers data
 - [C# / Lidgren.Network](https://github.com/lidgren/lidgren-network-gen3) - Reliable UDP networking library (.NET).
 - [C# / LiteNetLib](https://github.com/RevenantX/LiteNetLib) - Lite reliable UDP networking library (.NET/Mono).
 - [C# / PhotonEngine](https://photonengine.com) - Hybrid multiplayer game networking platform (SaaS, Cloud).
+- [C# / Telepaty](https://github.com/vis2k/Telepathy) - Simple, message based, MMO Scale TCP networking in C#. And no magic.
```

Comment:
I'm not a fan of this project just because of these quotes from author:

> Telepathy was originally designed for uMMORPG after 3 years in UDP hell.

> Async Sockets: didn't perform better in our benchmarks.

> UDP vs. TCP: Minecraft and World of Warcraft are two of the biggest multiplayer games of all time and they both use TCP networking. There is a reason for that.

I think it's fair to not have this project here since the repo aims to include "awesome" things.